app/main.py

```diff
@@ -7,6 +7,7 @@
 from app.database.database import engine, get_db
 from app.dependencies import (logger, MEDIA_PATH, STATIC_PATH, templates)
 from app.internal import daily_quotes, json_data_loader
+from app.internal.international_days import load_international_days
```

Comment:
Can you maybe use the `json_data_loader`? :)

Reply:
fixed